arch/risc-v/include/setjmp.h

```diff
@@ -25,13 +25,24 @@
  * Included Files
  ****************************************************************************/
 
+#include <nuttx/config.h>
+#include <nuttx/compiler.h>
+
 /****************************************************************************
  * Public Types
  ****************************************************************************/
 
 struct setjmp_buf_s
 {
   long regs[14];
+
+  /* Float callee register : fs0-fs11 */
+
+#ifdef CONFIG_ARCH_DPFPU
+  double fregs[12];
+#elif defined(CONFIG_ARCH_FPU)
```

Comment:
How do we handle `CONFIG_ARCH_QPFPU` case?

Reply:
Maybe we should add
```
#if defined(CONFIG_ARCH_QPFPU)
  long double fregs[12];
#elif defined(CONFIG_ARCH_DPFPU)
```

Reply:
BTW why do we store only 12 values? From `arch/risc-v/src/common/riscv_fpu.S` I see that there are 32 values? Are 20 left values preserved somehow?

Reply:
> How do we handle `CONFIG_ARCH_QPFPU` case?

Yes, since all other place handle CONFIG_ARCH_QPFPU correctly.

Reply:
> BTW why do we store only 12 values? From `arch/risc-v/src/common/riscv_fpu.S` I see that there are 32 values? Are 20 left values preserved somehow?

Only fs0-fs11 marked as `Callee`.